java/driver/flight-sql-validation/src/test/java/org/apache/arrow/adbc/driver/flightsql/FlightSqlQuirks.java

```diff
@@ -48,6 +50,8 @@ public AdbcDatabase initDatabase(BufferAllocator allocator) throws AdbcException
 
     final Map<String, Object> parameters = new HashMap<>();
     parameters.put(AdbcDriver.PARAM_URL, url);
+    parameters.put("adbc.username", System.getenv(FLIGHT_SQL_USER_ENV_VAR));
```

Comment:
As mentioned, let's use just "username" for consistency with other implementations